modules/product-reviews-magento/graphql/queries/CategoryDetail.ext.ts

```diff
@@ -0,0 +1,26 @@
+import field from '#ioc/graphql/field'
+import Review from '#ioc/graphql/fragments/Review'
+import CATALOG_REVIEWS_PER_PAGE from '#ioc/config/CATALOG_REVIEWS_PER_PAGE'
+import addFields from '#ioc/utils/graphql/addFields'
+
+export default (self: any) => {
+  return () => {
+    const query = self()
+
+    addFields(query, 'products.items', {
+      rating_summary: field(),
+      review_count: field(),
+      reviews: field()
```

Comment:
Why are you adding whole reviews on the category detail query? Thats redundant. rating summary and rating count should be enough